lib/validator.js

```diff
@@ -172,7 +172,7 @@ var EXTENDED_COLOR_KEYWORDS = {
 }
 
 var LENGTH_REGEXP = /^[-+]?\d*\.?\d+(\S*)$/
-var SUPPORT_CSS_UNIT = ['pt', 'wx']
+var SUPPORT_CSS_UNIT = ['pt', 'wx', 'px']
```

Comment:
Weex uses `px` as default unit and will convert it to number format. No need to add this.